testnet/stacks-node/src/tests/neon_integrations.rs

```diff
@@ -10417,3 +10420,93 @@ fn microblock_miner_multiple_attempts() {
 
     channel.stop_chains_coordinator();
 }
+
+#[test]
+#[ignore]
+fn test_submit_and_observe_peg_in_request() {
```

Comment:
While what you have here is great, I just wanted to let you know that there are lighter-weight operations available for testing burnchain operations.  For example, see `test_pox_btc_ops()` and `test_stx_transfer_btc_ops()` in `src/chainstate/coordinator/tests.rs`.

Reply:
Thanks! I'll study those test cases and see if it's worth complementing the existing tests with something more lightweight to further future-proof the code. Since the neon integration tests are slow, the more code paths we can cover in lighter tests the better.

Reply:
I added a test case for the peg-in operation in the `chainstate::coordinator::tests` now to complement the existing test cases.

Reply:
✅ Done